packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -565,6 +565,33 @@ the configured path by running this command: flutter config --android-studio-dir
     }
   }
 
+  static Version? _parseVersion(String text) {
+    // Example string fort EAP: EAP AI-242.21829.142.2422.12358220
+    // We try to capture 2422 here.
```

Comment:
From my earlier suggestion in https://github.com/flutter/flutter/pull/156293#discussion_r1797427098:

```suggestion
    // Matches the version string for Preview builds on macOS.
    // Example match: EAP AI-242.21829.142.2422.12358220
    // We try to capture "2422" here, which can be translated to a
    // more human-friendly "24.2.2".
```

Reply:
Yep, missed that, thanks.